modular_citadel/code/modules/reagents/chemistry/reagents/SDGF.dm

```diff
@@ -52,6 +52,7 @@ IMPORTANT FACTORS TO CONSIDER WHILE BALANCING
 	inverse_chem_val 		= 0.5
 	inverse_chem		= /datum/reagent/impure/SDZF
 	can_synth = TRUE
+	value = REAGENT_VALUE_EXCEPTIONAL
```

Comment:
while it does require uranium, i can still mass produce this stuff pretty quickly?

Reply:
I didn't know, thanks for pointing out.